src/pages/community.yaml

```diff
@@ -482,7 +482,7 @@ components:
         imgPlaceholder: false
         fullHeight: true
         rounded: true
-        url: https://medium.com/designers-italia/una-grammatica-dei-servizi-pubblici-digitali-7077aa1fc5ef
+        url: https://medium.com/designers-italia/progettare-con-qualita-accessibilita-al-centro-del-design-system-del-paese-5e3599170099
```

Comment:
@gianvitofanelli questa PR è obsoleta vero? Questo link in caso sarebbe sbagliato, tornerebbe indietro. Possiamo chiudera?

Reply:
Ormai obsoleta, sì